src/@ionic-native/plugins/purchases/index.ts

```diff
@@ -44,162 +185,30 @@ import { Observable } from 'rxjs';
  *      }
  *  }
  * ```
- * #### 3. Displaying Available Products
```

Comment:
Removed all this because it's hard to maintain and better to just link to the docs